models/agency-node.go

```diff
@@ -0,0 +1,55 @@
+package models
+
+import (
+	"cloud.google.com/go/bigquery"
+	"github.com/wopta/goworkspace/lib"
+	"time"
+)
+
+type AgencyNode struct {
+	Name               string                `json:"name" firestore:"name" bigquery:"name"`
+	VatCode            string                `json:"vatCode,omitempty" firestore:"vatCode,omitempty" bigquery:"vatCode"`
+	Phone              string                `json:"phone,omitempty" firestore:"phone,omitempty" bigquery:"phone"`
+	Address            *NodeAddress          `json:"address,omitempty" firestore:"address,omitempty" bigquery:"-"`
+	Manager            *AgentNode            `json:"manager,omitempty" firestore:"manager,omitempty" bigquery:"manager"`
+	RuiCode            string                `json:"ruiCode" firestore:"ruiCode" bigquery:"ruiCode"`
+	RuiSection         string                `json:"ruiSection" firestore:"ruiSection" bigquery:"ruiSection"`
+	RuiRegistration    time.Time             `json:"ruiRegistration" firestore:"ruiRegistration" bigquery:"-"`
+	BigRuiRegistration bigquery.NullDateTime `json:"-" firestore:"-" bigquery:"ruiRegistration"`
+	Skin               *Skin                 `json:"skin,omitempty" firestore:"skin,omitempty" bigquery:"-"`
+	Pec                string                `json:"pec,omitempty" firestore:"pec,omitempty" bigquery:"-"`
+	Website            string                `json:"website,omitempty" firestore:"website,omitempty" bigquery:"-"`
+}
+
+func (an *AgencyNode) Sanitize() {
+	an.Name = lib.ToUpper(an.Name)
+	an.VatCode = lib.ToUpper(an.VatCode)
+	an.Phone = lib.TrimSpace(an.Phone)
```

Comment:
should we add a level of normalization where we add the +39 as default if not present?